CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+## [Preview Release 5.0.0-preview1.xxx] - 2022-03-09
+
+### Added
+
+- Added SqlDataSourceEnumerator. [#1430](https://github.com/dotnet/SqlClient/pull/1430), [Read more](#sql-data-source-enumerator)
+- Added new attestation protocol `None` option to forgo enclave attestation when using VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425) and [#1419](https://github.com/dotnet/SqlClient/pull/1419)
+- Added a new AppContext switch to suppress insecure TLS warnings. [#1457](https://github.com/dotnet/SqlClient/pull/1457)
+
+### Fixed
+
+- Suppressed TLS security warning with `Encrypt = false` by new AppContext switch. [#1457](https://github.com/dotnet/SqlClient/pull/1457)
```

Comment:
Already present in the Added section.

Reply:
this is a change and a fix so we have them in both

Reply:
Well, technically everything is a change, but we're not going to have duplicate entries for everything. If it's a bugfix, it should only be in fix.

Reply:
This isn't a fix and could be removed.